### DISCUSSION TITLE
Kernel Freeze

### DISCUSSION THREAD

**Discussion by workinhd** [Joshua-Riek/ubuntu-rockchip#452] [General]
Is this needed if you download the prebuilt image?

**Comment by Joshua-Riek** [comment]
What?

**Reply by workinhd** [reply]
I saw in kernel instructions to run sudo apt-mark hold linux-image-5.10.160-rockchip linux-headers-5.10.160-rockchip.  Is that only needed if you compile everything yourself?  I downloaded the 1.29 release for OrangePi5+
